src/config/http.rst

```diff
@@ -107,124 +107,111 @@ HTTP Server Options
         enabled on a per-request basis for any delayed JSON response call by adding
         ``?buffer_response=true`` to the request's parameters.
 
-.. config:section:: httpd :: HTTP Server Options
-
     .. config:option:: allow_jsonp :: Enables JSONP support
 
+        .. versionchanged:: 3.2
```

Comment:
We could indicate how it changed I think?


```
.. versionchanged:: 3.2 moved to `httpd` to `chttpd` section
```

Would that render properly I wonder?